src/lib/homestore_backend/index_kv.hpp

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+#include <homestore/btree/btree_kv.hpp>
+#include <homestore/index/index_table.hpp>
+#include <homestore/index/index_internal.hpp>
+#include <homestore/index_service.hpp>
+#include <homestore/blk.h>
+#include "lib/blob_route.hpp"
+
+namespace homeobject {
+
+class BlobRouteKey : public homestore::BtreeKey {
```

Comment:
I think `BlobRouteKey` can _be_ both a `BtreeKey` and a `BlobRoute` so you can remove the `friend` yeah?

Reply:
Discussed with Brian. Both 'has a' or 'is a' is ok in this case.